### PR TITLE
more reprod builds

### PR DESCRIPTION
**Pull request by My1**
I signed all I could build so far including the tagged but not yet asserted 921, if it doesnt fit, I'll just remove it.

### CODE REVIEW

File: releases/firmware-btc-v9.2.1/assertion.txt [hunk 1]
```diff
@@ -0,0 +1,8 @@
+By signing this file, the signer confirms that the firmware binary built from:
+
+git tag firmware-btc-only/v9.2.1
+git commit hash e8de37ec83379c4ae55b346333a14fb98d4ebde4
+
+resulted, at the time of signing, in a firmware binary file with the following sha256sum:
+
+d9eda2376d776f94b96fee0399b4df0105299addcd94dd0eec2cc17e05052bef
```
Comment (x1ddos) [inline releases/firmware-btc-v9.2.1/assertion.txt:8]: This is the hash @benma and I come up with:

    b3a59c1c758f7cf0f29403fdfc939ffa510e3057a9b8f3411ccab05ac132b56e

File: releases/firmware-v9.2.1/assertion.txt [hunk 2]
```diff
@@ -0,0 +1,8 @@
+By signing this file, the signer confirms that the firmware binary built from:
+
+git tag firmware/v9.2.1
+git commit hash e8de37ec83379c4ae55b346333a14fb98d4ebde4
+
+resulted, at the time of signing, in a firmware binary file with the following sha256sum:
+
+f31f0fcacadd2813413caf2f723c49c56a1a4d052aac3526f87e0b4141a01101
```
Comment (x1ddos) [inline releases/firmware-v9.2.1/assertion.txt:8]: @benma's and my build results in this hash:

    0100cc2fc39896738026a48c5030c0973852d1cdc173f14246cb156bb91db133

Are you sure you didn't have a dirty state? When you build, cmake outputs versions at the beginning. Check they don't contain `+dirty` flag.

**Comment by x1ddos** [comment]
> more reprod builds

We usually prefix commit title with a contex. If you see history in the `releases/` dir, it's almost ways `releases: ...` - https://github.com/digitalbitbox/bitbox02-firmware/commits/master/releases

Would be nice to have this prefix here too.

**Comment by My1** [comment]
okay, got a build with the hashes too, no idea what crapped here but maybe I was just a bit too hasty last night.

**Comment by My1** [comment]
okay all done.

**Comment by x1ddos** [comment]
:heavy_check_mark: thank you!

**Review by x1ddos — APPROVED** [review]

**Pull request merged**